org/skypro/skyshop/article/Article.java

```diff
@@ -0,0 +1,47 @@
+package org.skypro.skyshop.article;
+
+import org.skypro.skyshop.searchable.Searchable;
+
+public class Article implements Searchable {
+
+    private final String articleTitle;
+    private final String textOfTheArticle;
+
+    public Article (String articleTitle, String textOfTheArticle){
+        this.articleTitle = articleTitle;
+        this.textOfTheArticle = textOfTheArticle;
+    }
+
+    public String getArticleTitle(){
+        return articleTitle;
+    }
+
+    public String getTextOfTheArticle(){
+        return textOfTheArticle;
+    }
+
+    public String toString(){
+        return "Название статьи: " + articleTitle +
+                " Текст статьи: " + textOfTheArticle;
```

Comment:
см. внимательнее задание
```suggestion
        return articleTitle + "\n" + textOfTheArticle;
```